{
	"id": "CVE-2017-9814",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2017-9814",
				"description": "cairo-truetype-subset.c in cairo 1.15.6 and earlier allows remote attackers to cause a denial of service (out-of-bounds read) because of mishandling of an unexpected malloc(0) call.",
				"references": [
					{
						"source": "CVE",
						"url": "http://lists.opensuse.org/opensuse-security-announce/2020-07/msg00042.html"
					},
					{
						"source": "CVE",
						"url": "https://bugs.freedesktop.org/show_bug.cgi?id=101547"
					},
					{
						"source": "CVE",
						"url": "https://lists.apache.org/thread.html/rf9fa47ab66495c78bb4120b0754dd9531ca2ff0430f6685ac9b07772%40%3Cdev.mina.apache.org%3E"
					},
					{
						"source": "CVE",
						"url": "https://security.gentoo.org/glsa/201904-01"
					},
					{
						"source": "mitre",
						"url": "http://lists.opensuse.org/opensuse-security-announce/2020-07/msg00042.html"
					},
					{
						"source": "mitre",
						"url": "https://bugs.freedesktop.org/show_bug.cgi?id=101547"
					},
					{
						"source": "mitre",
						"url": "https://lists.apache.org/thread.html/rf9fa47ab66495c78bb4120b0754dd9531ca2ff0430f6685ac9b07772%40%3Cdev.mina.apache.org%3E"
					},
					{
						"source": "mitre",
						"url": "https://security.gentoo.org/glsa/201904-01"
					}
				],
				"modified": "2024-08-05T17:18:01.936Z"
			}
		}
	],
	"data_source": {
		"id": "mitre-v5",
		"raws": [
			"vuls-data-raw-mitre-v5/2017/CVE-2017-9814.json"
		]
	}
}
